static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -58,6 +60,95 @@ function ActionSet({
   const canMarkReviewed =
     anySelected && (allInQuerySelected || selectedIssues.some(issue => !!issue?.inbox));
 
+  // Determine whether to nest "Merge" and "Mark as Reviewed" buttons insde
+  // the dropdown menu based on the current screen size
+  const theme = useTheme();
+  const nestMergeButton = useMedia(`(max-width: ${theme.breakpoints[2]})`);
```

Comment:
sweet!

Reply:
What would make it expensive?

Reply:
hmm I think this should be pretty inexpensive (?) its value only changes when `matchMedia().matches` changes, which happens only once every time we cross the breakpoint threshold.

Reply:
Yeah I think so too

Reply:
fair enough... I asked because in the past I had some issues with this solution